src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -21,16 +21,16 @@
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
-    /// The ConvertTo-Json command
-    /// This command convert an object to a Json string representation
+    /// The ConvertTo-Json command.
+    /// This command convert an object to a Json string representation.
```

Comment:
Use the plural 'converts' versus 'convert'

Reply:
Fixed.